plugins/metadata/lib/model/IMetadataPeer.php

```diff
@@ -9,4 +9,6 @@
 	public static function retrieveByPK($pk, PropelPDO $con = null);
 	
 	public static function validateMetadataObjects($profileField, $objectIds, &$errorMessage);
+
+	public static function getEntry($objectId);
```

Comment:
kMetadataManager::getObjectPeer returns an instance of an object
in PHP you can call static methods on object instances. 
I need the object itself to get the right function.

we also do it in kMetadataManager::validateMetadataObjects ($objectPeer->validateMetadataObjects)